packages/agents-bot-hosting/src/auth/authProvider.ts

```diff
@@ -5,6 +5,15 @@
 
 import { AuthConfiguration } from './authConfiguration'
 
+/**
+ * Interface representing an authentication provider.
+ */
 export interface AuthProvider {
+  /**
+   * Gets an access token for the specified authentication configuration and scope.
+   * @param authConfig - The authentication configuration.
```

Comment:
missing the {types}

Reply:
my bad, TSDoc does not require {types}

https://typedoc.org/documents/Tags._param.html#tsdoc-compatibility